### ISSUE TITLE
Clothing1M Performance

### ISSUE THREAD

**Issue by botanyldrm** [nazmul-karim170/UNICON-Noisy-Label#4]
Hello. I have run your repository directly  over Clothing1M to reproduce results. I have followed your following parameter table: 

![image](https://user-images.githubusercontent.com/76845506/175812922-f88a3020-5215-4862-99ee-a09d7e30eba8.png)

I have run Clothing1M train script with following command by looking parameter table(I set only settable parameters):

`python ./Train_clothing1M.py \
--data_path /truba/scratch/byildirim/clothing1M \
--learning_rate 0.002 \
--alpha 0.5 \
--batch_size 32 \
--num_epochs 8 \
--T 0.5 \
--lambda_c 0.025 \
`

Then, I have obtained following result:

    Validation Epoch:0  Acc1:65.37  Acc2:67.51
    Epoch:0 	  Acc: 68.38% (96.27%) 
    Validation Epoch:1  Acc1:69.08  Acc2:69.28
    Epoch:1 	  Acc: 70.44% (97.33%) 
    Validation Epoch:2  Acc1:69.78  Acc2:70.57
    Epoch:2 	  Acc: 71.33% (97.49%) 
    Validation Epoch:3  Acc1:70.75  Acc2:71.49
    Epoch:3 	  Acc: 72.43% (97.53%) 
    Validation Epoch:4  Acc1:70.87  Acc2:71.48
    Epoch:4 	  Acc: 72.28% (97.09%) 
    Validation Epoch:5  Acc1:71.11  Acc2:70.52
    Epoch:5 	  Acc: 72.28% (97.46%) 
    Validation Epoch:6  Acc1:72.05  Acc2:70.73
    Epoch:6 	  Acc: 72.54% (97.50%) 
    Validation Epoch:7  Acc1:72.05  Acc2:71.25
    Epoch:7 	  Acc: 72.54% (97.50%) 
    Validation Epoch:8  Acc1:72.05  Acc2:71.54
    Epoch:8 	  Acc: 72.54% (97.50%) 

Here, the best result is %72.54 which is 2.04 lower than the reported one. Do I miss something or do something wrong? Is there and preprocessing step which is not mentioned or something different from hyperparameter table?

Also, in Train_clothing1M.py, default value of **num_batches** is set to 1000 and this was not mentioned in parameter table. Setting this to 1000 and batch size to 32 decreased number of observed sample number to 32000 during a single epoch. Am I wrong?
